src/khiva/dimensionality.cpp

```diff
@@ -10,15 +10,24 @@
 #include <algorithm>
 #include <boost/math/distributions/normal.hpp>
 #include <cmath>
-#include <iostream>
 #include <iterator>
 #include <limits>
 #include <stdexcept>
 #include <utility>
 #include <vector>
+#include <map>
+#include <set>
 
-float computeTriangleArea(khiva::dimensionality::Point a, khiva::dimensionality::Point b,
-                          khiva::dimensionality::Point c) {
+namespace khiva {
```

Comment:
Is not cleaner to just `using namespace khiva::dimensionality`?

Reply:
Yes, sure!